src/PKSim.Presentation/Presenters/Simulations/PopulationPKAnalysisPresenter.cs

```diff
@@ -34,33 +40,67 @@ public class PopulationPKAnalysisPresenter : PKAnalysisPresenter<IPopulationPKAn
       {
          _pkAnalysesTask = pkAnalysesTask;
          _exportTask = exportTask;
-         _allAnalyses = new List<PopulationPKAnalysis>();
+         _allAnalysesOnCurves = new List<PopulationPKAnalysis>();
+         _allAnalysesOnIndividuals = new List<PopulationPKAnalysis>();
          _populationPKAnalysisToDTOMapper = populationPKAnalysisToDTOMapper;
          _globalPKAnalysisPresenter = globalPKAnalysisPresenter;
          AddSubPresenters(_globalPKAnalysisPresenter);
          _view.AddGlobalPKAnalysisView(_globalPKAnalysisPresenter.View);
       }
 
-      public void CalculatePKAnalysis(IPopulationDataCollector populationDataCollector, ChartData<TimeProfileXValue, TimeProfileYValue> timeProfileChartData)
+      public void CalculatePKAnalysisOnCurves(IPopulationDataCollector populationDataCollector, ChartData<TimeProfileXValue, TimeProfileYValue> timeProfileChartData)
       {
-         _allAnalyses.Clear();
+         CalculatePKAnalysis(populationDataCollector, _allAnalysesOnCurves, _pkAnalysesTask.CalculateFor(populationDataCollector, timeProfileChartData), _allPKAnalysesOnCurves);
+      }
+
+      public void CalculatePKAnalysisOnIndividuals(IPopulationDataCollector populationDataCollector, IEnumerable<PopulationPKAnalysis> pks)
+      {
+         CalculatePKAnalysis(populationDataCollector, _allAnalysesOnIndividuals, pks, _allPKAnalysesOnIndividuals);
+      }
+
+      private bool _pkAnalysisOnIndividualsEnabled = true;
+
+      public bool PKAnalysisOnIndividualsEnabled { 
+         get
+         {
+            return _pkAnalysisOnIndividualsEnabled;
+         }
+         set
+         {
+            _pkAnalysisOnIndividualsEnabled = value;
+            View.EnablePKAnalysisOnIndividualsTab(value);
+         }
+      }
+
+      private void CalculatePKAnalysis(IPopulationDataCollector populationDataCollector, List<PopulationPKAnalysis> allAnalysis, IEnumerable<PopulationPKAnalysis> sourcePKs, List<PopulationPKAnalysis> targetPKs)
+      {
+         allAnalysis.Clear();
          _populationDataCollector = populationDataCollector;
-         _allPKAnalyses = _pkAnalysesTask.CalculateFor(populationDataCollector, timeProfileChartData);
-         _allAnalyses.AddRange(_allPKAnalyses);
+         targetPKs.Clear();
+         targetPKs.AddRange(sourcePKs);
```

Comment:
Do not use PK. You are using the same acronyms for PK Parameter, PK analyses, which is in both cases wrong
PK means PharmacoKinetic.